lumigator/frontend/src/types/JobDetails.ts

```diff
@@ -5,7 +5,17 @@ export type JobDetails = {
   submission_id: string
   driver_info?: unknown
   status: WorkflowStatus
-  entrypoint: string
+  config: {
+    name: string
+    dataset?: {
+      id: string
+      name: string
+    }
+    max_samples?: number
+    model?: {
+      path: string
+    }
```

Comment:
when can these be `undefined` ? I'm guessing we typically show them in the UI, so ideally they should be there.

Is this one of the cases where the data structure for "annotation jobs" are different?

Reply:
I've updated this, only dataset might be missing (e.g. for evaluation jobs)